plugins/woocommerce/tests/e2e-pw/tests/merchant/create-post.spec.js

```diff
@@ -0,0 +1,59 @@
+const { test, expect, request } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+const postTitle = `Post-${ new Date().getTime().toString() }`;
+
+test.describe( 'Can create a new post', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterAll( async ( { baseURL } ) => {
+		const base64auth = btoa( `${ admin.username }:${ admin.password }` );
+		const wpApi = await request.newContext( {
+			baseURL: `${ baseURL }/wp-json/wp/v2/`,
+			extraHTTPHeaders: {
+				Authorization: `Basic ${ base64auth }`,
+			},
+		} );
+		let response = await wpApi.get( `posts` );
+		const allPosts = await response.json();
+
+		await allPosts.forEach( async ( post ) => {
+			if ( post.title.rendered === postTitle ) {
+				response = await wpApi.delete( `posts/${ post.id }` );
+				expect( response.ok() ).toBeTruthy();
+			}
+		} );
+	} );
+
+	test( 'can create new post', async ( { page } ) => {
+		await page.goto( 'wp-admin/post-new.php', {
+			waitUntil: 'networkidle',
+		} );
+
+		await page
+			.getByRole( 'textbox', { name: 'Add Title' } )
+			.fill( postTitle );
+
+		await page.getByRole( 'button', { name: 'Add default block' } ).click();
+
+		await page
+			.getByRole( 'document', {
+				name:
+					'Empty block; start writing or type forward slash to choose a block',
+			} )
+			.fill( 'Test Post' );
+
+		await page
+			.getByRole( 'button', { name: 'Publish', exact: true } )
+			.click();
+
+		await page
+			.getByRole( 'region', { name: 'Editor publish' } )
+			.getByRole( 'button', { name: 'Publish', exact: true } )
+			.click();
+
+		expect(
+			await page.getByText( `${ postTitle } is now live.` )
+		).toBeTruthy();
```

Comment:
Might be better to use the `.toBeVisible()` assertion here: https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible

My understanding (and it may be wrong) is that `.toBeTruthy()` will return true if the element is present in the DOM, but it doesn't necessarily need to be visible on the page (it could be hidden).  Not a big deal here because the message is rendered after the post is made, but some pages may keep messaging present and just hide/un-hide it as needed.

Reply:
Noted. Thank you.